packages/util/src/signature.ts

```diff
@@ -39,6 +39,21 @@ function calculateSigRecovery(v: bigint, chainId?: bigint): bigint {
   return v - (chainId * BigInt(2) + BigInt(35))
 }
```

Comment:
I guess we can (and also: should) delete this method since this is redundant now.

Reply:
By deleting this method, I had to shift `ecrecover` to use the new `calculateSigRecoveryFromV()` instead of the deleted `calculateSigRecovery()`

`calculateSigRecovery` had a clause that if the `chainId` parameter was undefined, the method would throw.  Without chainId as a parameter at all, `calculateSigRecoveryFromV()` will always return.

Several tests in util and tx relied on this method throwing an error.

Reply:
If this causes any problems than in  doubt *keep* the method. This was just a quick shot from me assuming this internal method is not needed any more.